substrate/frame/contracts/src/lib.rs

```diff
@@ -250,9 +250,10 @@ pub mod pallet {
 	#[pallet::storage_version(STORAGE_VERSION)]
 	pub struct Pallet<T>(_);
 
-	#[pallet::config]
+	#[pallet::config(with_default)]
 	pub trait Config: frame_system::Config {
 		/// The time implementation used to supply timestamps to contracts through `seal_now`.
+		#[pallet::no_default]
```

Comment:
Not possible to provide a default here?

Reply:
Yeah you would most likely want to override it, if you do anything that use that type in your tests, but I can provide an implementation that is unimplented!